modules/nf-core/samtools/import/main.nf

```diff
@@ -0,0 +1,60 @@
+process SAMTOOLS_IMPORT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::samtools=1.17"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.17--h00cdaf9_0':
+        'biocontainers/samtools:1.17--h00cdaf9_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*.sam") , emit: sam,     optional: true
+    tuple val(meta), path("*.bam") , emit: bam,     optional: true
+    tuple val(meta), path("*.cram"), emit: cram,    optional: true
```

Comment:
I'll pass on the alphabetical order for once

Reply:
It's in order of compression 😅